source/_integrations/airthings_ble.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Airthings BLE
+description: Instructions on how to set up Airthings Devices over Bluetooth LE.
+ha_category:
+  - Environment
+  - Sensor
+ha_release: '2022.9'
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@vincegio'
+ha_domain: airthings_ble
+ha_bluetooth: true
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+Integrates Airthings BLE sensors into Home Assistant.
+
+## Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant.
+
+It is good if you know the MAC address of your Airthings Device. As of writing, the Airthings app does not show you the address of a device. 
```

Comment:
Why is it important to find out the MAC address?

Reply:
I guess knowing the MAC address during setup is only important if you have more then one Airthings device. The MAC address is part of the auto generated device name, to make it unique and possible to differentiate the devices. To avoid over complicating this concern in the docs, specially because most users would just sit on one single device (I guess), I would make this as own chapter in the docs and inform the user with more then one device to check the MAC address labeled on the device.

Reply:
Actually, appears that it is not. I found that the serial number is given to us so can just point to where to find this no. on the device :)

Reply:
You are damn right, @vincegio ! Now I remember all the struggle I had finding the MAC address of mine Airthings device, when using the custom component [airthings_wave](https://github.com/custom-components/sensor.airthings_wave) which could not auto detect my device.

Reply:
Yeah, should be easier with just S/N hehe